providers/aws/resources/aws.lr.manifest.yaml

```diff
@@ -3011,46 +3011,15 @@ resources:
     platform:
       name:
       - aws
-  aws.timestream:
```

Comment:
???

Reply:
That was an old generation. The root switched to aws.timestream.liveanalytics